src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+}
+
+#block1 {
+  background-color: red;
+  height: 100px;
+}
+
+#block2 {
+  background-color: green;
+  height: 100px;
+}
+
+#block3 {
+  background-color: blue;
+  height: 100px;
+}
```

Comment:
IDs are used for JS, try not to style the id if you can avoid it. This makes impossible to reuse the code. Stylize classes. [Style guide](https://mate-academy.github.io/style-guides/htmlcss.html#id-selectors)

Reply:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.